oak/server/rust/oak_glue/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "oak_glue"
+version = "0.1.0"
+authors = ["David Drysdale <drysdale@google.com>"]
+edition = "2018"
+license = "Apache-2.0"
```

Comment:
That would be possible, but I deliberately made this a separate crate to make sure this would be easy to reverse and/or remove when the C++->Rust linkage flips to a Rust->C++ linkage.